### DISCUSSION TITLE
long press video as 3-dot menu

### DISCUSSION THREAD

**Discussion by castdrian** [ReVanced/revanced-discussions#168] [Question]
Title. This used to briefly be possible at some point. Quite frankly the three dot button is very smol and the chance of missing it and having the video start playing is usually higher than hitting it and getting the menu to open (unless you have really tiny hands or a ginormous device)

**Comment by oSumAtrIX** [comment]
Many people want long press to do different things

**Comment by MulverineX** [comment]
Well if long press opens the three dot menu then the other things can be added to said menu. And/or it becomes an option
